@types/pal/image.d.ts

```diff
@@ -0,0 +1,87 @@
+declare module 'pal/image' {
+    export type ImageSource = import('pal/image/types').ImageSource;
+    export type IMemoryImageSource = import('pal/image/types').IMemoryImageSource;
+    export type PixelFormat = import('cocos/asset/assets/asset-enum').PixelFormat;
+    export class ImageData {
+        constructor (imageAsset?: ImageSource | ArrayBufferView);
+        /**
+         * Destroy resources.
+         */
+        destroy (): void;
+
+        /**
+         * Get the url of the image(when source is HTMLImageElement).
+         */
+        get src (): string;
+        /**
+         * Set the url of the image(when source is HTMLImageElement).
+         */
+        set src(url: string);
```

Comment:
```suggestion
        /**
         * Get and set the url of the image (when source is HTMLImageElement).
         */
        get src (): string;
        set src (url: string);
```

we only need one comment for getter/setter, the same with others